index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Ishita Keshawani" />
+    <meta name="description" content="Beauty E-commerce Store" />
+    <meta name="keywords" content="beauty store,e-commerce,react.js" />
+    <link
+      rel="stylesheet"
+      href="https://felizui.netlify.app/styles/index.css"
+    />
+    <link rel="stylesheet" href="/pages/homepage.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+      integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="icon" href="/assets/Stunners.png" />
+    <title>Stunners-Hub</title>
+  </head>
+
+  <body>
+    <nav class="navbar semibold-font-weight">
+      <a class="nav-icon-link nav-link link-no-style" href="">
+        <img class="logoimage" src="/assets/Stunners.png" alt="" />
+        <span class="small-fontsize">Stunners Hub</span>
+      </a>
+      <a class="nav-link link-no-style active" href="">Home</a>
+      <a class="nav-link link-no-style" href="">Products</a>
+      <div class="nav-last-items">
+        <div class="searchbar">
+          <i class="fa-solid fa-magnifying-glass search-icon"></i>
+          <input
+            type="text"
+            class="search-input"
+            placeholder="Search on Stunners Hub"
+          />
+        </div>
+        <a href="http://" class="link-no-style nav-link nav-icon-link">
+          <i class="fa-regular fa-user"></i>
+          <span class="small-fontsize">Login</span>
+        </a>
+        <a href="http://" class="link-no-style nav-link nav-icon-link">
+          <i class="fa-regular fa-heart"></i>
+          <span class="small-fontsize">Wishlist</span>
+        </a>
+        <a href="http://" class="link-no-style nav-link nav-icon-link">
+          <img
```

Comment:
extra space is wasted here, use pretttier

Reply:
I am using it but sometimes it is not working. Do you have any idea of other tools?